code/game/objects/items/hot_potato.dm

```diff
@@ -52,6 +52,7 @@
 	var/atom/location = loc
 	location.visible_message(span_userdanger("[src] [detonate_explosion? "explodes" : "activates"]!"), span_userdanger("[src] activates! You've ran out of time!"))
 	if(detonate_explosion)
+		guy.gib()
```

Comment:
The guy's name is probably not `guy` you have to fill in that yourself

Reply:
im a bit stupid my bad, i thought it being in blocks meant its exactly what you wanted

Reply:
Fair point but it was just pesudocode